File: Data Structures & Algorithms/max-consecutive-ones/submission-0.rs
impl Solution {
    pub fn find_max_consecutive_ones(nums: Vec<i32>) -> i32 {
        let mut max_ones = 0;
        let mut running_ones = 0;
        for num in nums {
            if num == 1 { running_ones += 1; }
            else {
                max_ones = max(max_ones, running_ones);
                running_ones = 0; 
            }
        }
        return max(max_ones, running_ones);
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums_hashset = set(nums)
        inits = [num for num in nums if num - 1 not in nums_hashset] 
        max_seq = 0
        for init in inits:
            end = init
            while end + 1 in nums_hashset:
                end += 1
            max_seq = max(end - init + 1, max_seq)
        return max_seq